website/docs/r/batch_pool.html.markdown

```diff
@@ -268,6 +268,8 @@ A `resource_file` block supports the following:
 
 * `storage_container_url` - (Optional) The URL of the blob container within Azure Blob Storage. This URL must be readable and listable using anonymous access; that is, the Batch service does not present any credentials when downloading the blob. There are two ways to get such a URL for a blob in Azure storage: include a Shared Access Signature (SAS) granting read and list permissions on the blob, or set the ACL for the blob or its container to allow public access.
 
+* `identity_reference` - (Optional) An identity reference from pool's user assigned managed identity list.
```

Comment:
what sort of identity is returned? and id? a guid? is it managed? could it be user or msi?
```suggestion
* `identity_id` - (Optional) An identity reference from pool's user assigned managed identity list.
```

Reply:
Done.